src/pages/Auth/Login.tsx

```diff
@@ -0,0 +1,154 @@
+import { useAuth } from "context/auth-context";
+import { useUser } from "context/user-context";
+import {
+  ChangeEventHandler,
+  MouseEventHandler,
+  useEffect,
+  useState,
+} from "react";
+import { Link, Location, useLocation, useNavigate } from "react-router-dom";
+import { loginService } from "services";
+import { EmailType, TextType } from "__types__/typeUtils/base.types";
+
+export const Login = () => {
+  type LoginDataType = {
+    email: EmailType;
+    password: TextType;
+  };
+  const location = useLocation();
+  const locationState = location.state as { from: Location };
+
+  const testCreds = {
+    email: "guest@xyz.com",
+    password: "abc123hh",
+  };
+  const [status, setStatus] = useState({ isLoading: false, error: "" });
```

Comment:
instead of an object create a string based status

Reply:
Do you mean like
```js
status='loading'
status='error'
```
?